Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -715,9 +716,14 @@ def SwapKeepAndDrop(l):
                             'keep *_hltGeneralTracks_*_*',
                             'keep *_hltInitialStepTrackSelectionHighPurity_*_*',
                             'keep *_hltHighPtTripletStepTrackSelectionHighPurity_*_*',
-                            'keep *_hltInitialStepTracksT4T5TCLST_*_*',
+                            'keep *_hltInitialStepTracksT5TCLST_*_*',
                             'keep *_hltOfflinePrimaryVertices_*_*',
                         ])
+alpakaValidationLST.toModify(FEVTDEBUGHLTEventContent,
```

Comment:
```suggestion
(trackingLST & alpakaValidationLST).toModify(FEVTDEBUGHLTEventContent,
```

should this be coupled with `trackingLST`?